platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SystemExitUtils.java

```diff
@@ -74,6 +74,13 @@ public static void exitSystem(@NonNull final SystemExitCode exitCode, @Nullable
         exitSystem(exitCode, message, false);
     }
 
+    /**
+     * Same as {@link #exitSystem(SystemExitCode, String)}, but with a reversed order of parameters.
+     */
+    public static void exitSystem(@Nullable final String message, @NonNull final SystemExitCode exitCode) {
```

Comment:
I'm totally against that pattern of having 2 methods that do the same but only have a different order of params. What is the idea behind that?

Reply:
yeah, its bad, i just did it as a temp solution to limit the scope of the refactor. let me see if there is a simple way to avoid this

Reply:
removed